solidity/dashboard/src/components/TokenGrantOverview.jsx

```diff
@@ -78,23 +82,114 @@ export const TokenGrantUnlockingdDetails = ({
   selectedGrant,
   hideReleaseTokensBtn = false,
 }) => {
-  const { yourAddress, grantContract } = useContext(Web3Context)
+  const { yourAddress, grantContract, tokenStakingEscrow } = useContext(
+    Web3Context
+  )
   const showMessage = useShowMessage()
+  const closeMessage = useCloseMessage()
 
-  const releaseTokens = async (onTransactionHashCallback) => {
+  const releaseTokens = async (
+    onTransactionHashCallback,
```

Comment:
We use `openConfirmationModal`. I've refactored the modal approach in the dapp a bit in https://github.com/keep-network/keep-core/commit/e47b03253e541a375c43062a3db2d50ea220d3c7 , so it will probably fix this.